<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-Sandeep" name="Test" tests="3" failures="1" timestamp="17 Jul 2017 11:02:05 GMT" time="4.694" errors="0">
  <testcase name="doLogin" time="2.259" classname="DD_TestCases.LoginTest">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="0.911" classname="DD_TestCases.LoginTest">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="0.886" classname="DD_TestCases.LoginTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:92)
	at org.junit.Assert.fail(Assert.java:100)
	at DD_TestCases.LoginTest.doLogin(LoginTest.java:39)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Test -->
